config/index.js

```diff
@@ -1,4 +1,7 @@
+const swaggerConfig = require('./swagger-config');
+
 module.exports = {
   secret:
-        process.env.NODE_ENV === 'production' ? process.env.SECRET : 'secret'
+        process.env.NODE_ENV === 'production' ? process.env.SECRET : 'secret',
+  options: swaggerConfig.options
```

Comment:
`options` as a key here is not descriptive enough. `swaggerConfigOptions` best describe this.